routes/teams_routes.js

```diff
@@ -0,0 +1,96 @@
+'use strict';
+
+var Team = require('../models/Team');
+var bodyparser = require('body-parser');
+
+module.exports = function (router) {
+  router.use(bodyparser.json());
+
+  router.get('/teams', function (req, res) {
+    Team.find({}, function (err, data) {
+      if (err) {
+        console.log(err);
+        return res.status(500).json({msg: 'internal server error'});
+      }
+
+      res.json(data);
+    });
+  });
+
+  router.post('/teams/addteam', function (req, res) {
+    //create new instance of Team from the info entered in body response
+    var newTeam = new Team(req.body);
+    // use save method on newTeam to save data entered in body to database
+    newTeam.save(function (err, data) {
+      if (err && req.body.wins + req.body.losses < 162) {
+        console.log(err);
+        return res.status(500).json({msg: 'internal server error'});
+      }
+      if (err && req.body.wins + req.body.losses > 162) {
+        console.log(err);
+        return res.status(406).json({msg: 'team cannot have more than 162 games'});
+      }
+
+      res.json(data);
+    })
+  })
+
+  router.put('/teams/updateteam/:id', function (req, res) {
+    var updatedTeam = req.body;
+    Team.find({_id: req.params.id}, function (err, data) {
```

Comment:
The find method does not throw an error if there is no data matching the parameters. It simply returns an empty array. The query is not needed.